CHANGELOG.md

```diff
@@ -1,7 +1,43 @@
 ## Parse Server Changelog
 
 ### master
-[Full Changelog](https://github.com/ParsePlatform/parse-server/compare/2.6.5...master)
+[Full Changelog](https://github.com/ParsePlatform/parse-server/compare/2.7.0...master)
```

Comment:
Should change **ParsePlatform** to **parse-community** in the url, saves a 301 redirect.